test/unit/template_test.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class TemplateTest < MiniTest::Test
+  def test_serialize
```

Comment:
I'm not sure how to (or need to) further improve test coverage.

Reply:
I think if we add this to the benchmarks, and run those as part of CI it would add coverage for free.

We also need to make sure `Liquid::Template.load(serialize).render!` is, and will remain faster than `Liquid::Template.parse(source).render!`.

Reply:
I implemented benchmarks in Shopify/liquid#1385. Deserialize is about 20% faster than parsing from source.